modules/gapi/samples/onevpl_infer_single_roi.cpp

```diff
@@ -196,8 +196,8 @@ int main(int argc, char *argv[]) {
     std::string file_path = cmd.get<std::string>("input");
     const std::string output = cmd.get<std::string>("output");
     const auto face_model_path = cmd.get<std::string>("facem");
-    const auto streaming_queue_capacity = cmd.get<uint64_t>("streaming_queue_capacity");
-    const auto source_queue_capacity = cmd.get<uint64_t>("frames_pool_size");
+    const auto streaming_queue_capacity = cmd.get<size_t>("streaming_queue_capacity");
+    const auto source_queue_capacity = cmd.get<size_t>("frames_pool_size");
```

Comment:
> `.get<size_t>()`

Not all types are supported in `CommandLineParser`.
Here we can use `.get<int>` + check>=0 + cast

Reply:
thanks
i've faced with this too... added one more commit with implicit type expanding (uint32-> ...) for x86_64 system